frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -127,7 +127,10 @@ const ErrorRetrievingMessage: React.SFC<CVStatusMessageProps> = ({ cv }) => {
   ) : (
     <Tooltip content={truncateMiddle(retrievedUpdatesCondition.message, { length: 256 })}>
       <span>
-        <RedExclamationCircleIcon /> Error retrieving
+        <RedExclamationCircleIcon />{' '}
+        {retrievedUpdatesCondition.reason === 'VersionNotFound'
+          ? 'Version not found'
+          : 'Error retrieving'}
```

Comment:
I'd suggest just always showing `retrievedUpdatesCondition.reason`

Reply:
I didn't do that as we humanize in all other cases.  Do we want to remove all humanization?  Or let this be an outlier?